frontend/src/queries/nodes/DataVisualization/Components/Variables/Variables.tsx

```diff
@@ -224,6 +234,16 @@ const VariableInput = ({
                         >
                             {variableAsHogQL}
                         </code>
+                        <LemonSwitch
+                            size="xsmall"
+                            label="Null"
```

Comment:
Minor nit, `null` > `Null` - seeing null capitalized feels weird?! Also, do we want the label to be more like `"Set to null"` instead to make it more clear what it means? I imagine some people may think we're printing out a rogue `null` value to the UI and have no idea what this toggle is for